source/2019-11-29-the-ember-times-issue-126.md

```diff
@@ -65,13 +66,17 @@ If you can contribute to everyone's learning today, we ask that you help **updat
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [More Improvements Coming to Ember Inspector Soon 🚀](https://github.com/emberjs/ember-inspector/pull/1088)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+A huge shoutout and thank you are owed to [Godfrey Chan @chancancode](https://github.com/chancancode) and [Robert Wagner @rwwagner90](https://github.com/rwwagner90), the maintainer of the Ember Inspector, and everyone else involved for all their amazing work in getting the new and improved **Ember Inspector** ready for Octane! 🔥🔥🔥
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+With the release of Octane coming soon it is welcomed news that Octane's Glimmer components work in Ember Inspector now, in both the component tree and the object inspector. Other improvements include revamping the component hover inspection tool to match Chrome's inspection style, many bug fixes and a complete rewrite of the component tree logic that is more future proof.
+
+[Godfrey Chan @chancancode](https://github.com/chancancode), in particular, is owed a big thanks for helping design the `captureRenderTree` API in Ember that was needed to provide the Inspector with the information it needs and for providing a polyfill, so it works seamlessly across versions.
```

Comment:
For clarity, I might break this sentence into two. Something like
```suggestion
[Godfrey Chan @chancancode](https://github.com/chancancode), in particular, is owed a big thanks for helping design `captureRenderTree`. This API provides the Inspector the information needed to create a polyfill so that it works seamlessly across different versions of Ember.
```
?

Reply:
Going to leave this as is to make sure we're describing the functionality accurately.